jsk_2013_04_pr2_610/launch/demo_all.launch

```diff
@@ -1,7 +1,9 @@
 <launch>
   <arg name="plan" default="true"/>
+  <arg name="use_ffha" default="false" />
```

Comment:
@sktometometo why we have to set `use_ffha` false?

Reply:
This is because the original demo launch use downward planner instead of ffha.

https://github.com/jsk-ros-pkg/jsk_demos/blob/f677f804709c6695535e37b9e6a491997a118e21/jsk_2013_04_pr2_610/launch/demo_common.launch#L4-L5